content/docs/aro/cross-tenant-encryption-keys/_index.md

```diff
@@ -0,0 +1,335 @@
+networking components.  This is desirable in large enterprises due to separation of concerns 
+and limiting areas of control for groups to a small scope.  This does present challenges, as 
+those different groups must be able to integrate with one another.  Often times, the integration 
+is difficult, complex, and confusing.  This document serves as a way to clear up some of 
+the confusion by walking you through scenarios for cross-tenancy in ARO.
+
+This guide covers the below use case.  Feel free to ignore sections not related to your
+use case:
+
+- Use Encryption Keys that resides within a separate Azure subscription
+
+
+## Environment Setup
+
+We need to set the following inputs related to the tenant where ARO and its 
+networking is being installed before proceeding:
+
+* `SCRATCH_DIR` - a location from your workstation where temporary files are stored
+* `ARO_CLUSTER_NAME` - the name of the cluster you are creating (also used as the prefix for other resources)
+* `ARO_PULL_SECRET` - the pull secret to use when provisioning the OpenShift cluster
+* `ARO_RESOURCE_GROUP` - the resource group where the ARO cluster will be created
+* `ARO_LOCATION` - the location where the ARO cluster will be created
+* `ARO_VNET_RESOURCE_GROUP` - the resource group where the VNET for the ARO cluster will be created
+* `ARO_VNET` - the VNET where the ARO cluster will be created
+* `ARO_CONTROL_PLANE_SUBNET` - the subnet which hosts the ARO control plane
+* `ARO_WORKER_SUBNET` - the control plane subnet which hosts the ARO worker nodes
+* `ARO_APP_NAME` - the application name which represents the ARO identity which is 
+used to access resources in other tenants.
+
+We need to set the following inputs related to the tenant where the Azure Key Vault
+exists before proceeding:
+
+* `KV_RESOURCE_GROUP` - the resource group where the Azure Key Vault will be created
+* `KV_LOCATION` - the location where the Azure Key Vault will be created
+* `KV_VAULT_NAME` - the name of the Azure Key Vault that will be created
+* `KV_KEY_NAME` - the name of the key within the Azure Key Vault that will be created
+
+Feel free to use the sensible defaults below if you need to quickly see how this works:
+
+```bash
+SCRATCH_DIR=/tmp/scratch
+
+ARO_CLUSTER_NAME=multi-tenant
+ARO_PULL_SECRET=~/.azure/aro-pull-secret.txt
+ARO_RESOURCE_GROUP=${ARO_CLUSTER_NAME}-rg
+ARO_LOCATION=eastus
+ARO_VNET_RESOURCE_GROUP=${ARO_CLUSTER_NAME}-vnet-rg
+ARO_VNET=${ARO_CLUSTER_NAME}-vnet
+ARO_CONTROL_PLANE_SUBNET=${ARO_CLUSTER_NAME}-control-plane-subnet
+ARO_WORKER_SUBNET=${ARO_CLUSTER_NAME}-worker-subnet
+ARO_APP_NAME=${ARO_CLUSTER_NAME}-aro
+
+KV_RESOURCE_GROUP=${ARO_CLUSTER_NAME}-kv-rg
+KV_LOCATION=eastus
+KV_VAULT_NAME=${ARO_CLUSTER_NAME}-vault
+KV_KEY_NAME=${ARO_CLUSTER_NAME}-key
+```
+
+## Cross-Tenant Encryption Setup
+
+This section covers the guide for cross-tenant encryption.
+
+
+### Setup the ARO Tenant
+
+1. Be sure to login the *the tenant that will host your **ARO cluster***:
+
+```
```

Comment:
tab these out so they appear indented in the rendered markdown so its clear they're part of the ordered task.

Reply:
also ```bash

Reply:
Fixed in https://github.com/rh-mobb/documentation/pull/414/commits/a6ebd6f903ebbb609d06d188cfdbbba95ee813a1